<fix>[qga_zwatch]: fix bug in zwatch_qga_monitor_vm

zstack-robot-2 opened the pull request:
Resolves: ZSV-9935

Change-Id: I696d776879616a676a707562786676687968796b

sync from gitlab !6225

MatheMatrix commented on kvmagent/kvmagent/plugins/qga_zwatch.py:
```diff
@@ -211,6 +215,14 @@ def zwatch_qga_monitor_vm(self, uuid, qga):
+                    except Exception as fe:
+                        logger.debug(str(fe))
+                        pass
+
```
**Comment from tao.yang:**

已补充，但except Exception只是为了预防出现异常导致方法错误

Commit:
Revert "<fix>[qga]: report qga state after starting zwatch monitor for new vm"

This reverts commit d98d2662c9afe9af77b65c5c68848b38422f6f2d.

Commit:
<fix>[qga]: set timeout and catch exception for guest_exec_zs_tools

1. extend the execution timeout

2. catch exception to avoid being processed by kvmagent and displaying
sensitive field

Resolves: ZSV-9935

Change-Id: I6671636d66617a6e786878716e636b7965647266

MatheMatrix commented:
**Comment from tao.yang:**

@coderabbitai review
